lib/engines/content_block_manager/test/unit/app/models/content_block_edition/document/document_filter_test.rb

```diff
@@ -0,0 +1,28 @@
+require "test_helper"
+
+class ContentBlockManager::DocumentFilterTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  describe "documents" do
+    describe "when no filters are given" do
+      it "returns live documents" do
```

Comment:
I wonder if it might be better if we unit test this functionality, rather than hitting the database directly. So rather than checking what's returned, we just stub `ContentBlock::Document` and check the right calls are made to it?